presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkRow.java

```diff
@@ -17,18 +17,44 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class SerializedPrestoSparkPage
+public class PrestoSparkRow
         implements Serializable
 {
+    private static final int INSTANCE_SIZE = Long.BYTES * 2 /* headers */
+            + Integer.BYTES /* partition */
+            + Integer.BYTES /* length */
+            + Long.BYTES /* bytes pointer */
+            + Long.BYTES * 2 /* bytes headers */
```

Comment:
bytes headers and bytes length should probably go into `sizeOfByteArray`?

Reply:
sizeOfByteArray is in the Slice library. It is not available here =\